internal/core/thirdparty/tantivy/tantivy-binding/src/index_writer.rs

```diff
@@ -103,6 +111,20 @@ impl IndexWriterWrapper {
         }
     }
 
+    pub fn add_json(&mut self, data: &str, offset: Option<i64>) -> Result<()> {
+        match self {
+            IndexWriterWrapper::V5(writer) => writer.add_json(data, offset),
```

Comment:
V5 is used for building index for milvus version using V5. do we need this logic for this new feature?